playbooks/roles/airship-deploy-ucp/tasks/main.yml

```diff
@@ -7,7 +7,8 @@
 
 - name: Set site path
   set_fact:
-    site_path: "{{ upstream_repos_clone_folder }}/airship/treasuremap/site/{{ socok8s_site_name }}"
+    site_path: "{{ upstream_repos_clone_folder }}/airship/treasuremap/site/{{ socok8s_site_name }}/"
+    local_temp_path: "{{ socok8s_workspace }}/{{ socok8s_site_name }}/"
```

Comment:
I think _local_ and _temp_ are bit redundant. What about `tmp_site_path`, `local_site_path` or something like that?

Reply:
local its to indicate that its local to the ansible run, so its on local-host

I did not want to have any confusion between loca as in locahost and local as in local to the remote node

Reply:
So you can just exclude `local_` completely and leave `tmp_site_path`?

Anyway, this is just me nitpicking, feel free to ignore

Reply:
changed it to workspace_site_path so its clearer...maybe?